test/chpldoc/linkage-spec/exportTest.doc.chpl

```diff
@@ -165,3 +169,15 @@ proc getName() param {
 
 export "test"+getName() proc withParamExternCName() {
 }
+
+export record externRecordNoComment { }
+
+/* Commenting externRecordComment */
+export record externRecordComment { }
+
+export "c_name1" record externRecordRenamedNoComment { }
```

Comment:
I would like to see either this test expanded or an additional test in the `test/chpldoc/classes` directory where the symbol has fields and methods with documentation comments